bothub/api/grpc/organization/serializers.py

```diff
@@ -0,0 +1,120 @@
+from django.db.models import F
+from django_grpc_framework import proto_serializers
+from rest_framework import serializers
+
+from bothub.authentication.models import User
+from bothub.common.models import Organization, OrganizationAuthorization
+from bothub.protos import organization_pb2
+
+
+class SerializerUtils(object):
```

Comment:
here you used Serializer at the beginning of the class, in the others you used it at the end of the name. It would be interesting to standardize